.gitignore

```diff
@@ -1,26 +1,19 @@
-# Build directory
-build/
-
-# Compiled class file
-*.class
-
-# Log file
-*.log
-
-# BlueJ files
-*.ctxt
-
-# Mobile Tools for Java (J2ME)
-.mtj.tmp/
-
-# Package Files #
-*.jar
-*.war
-*.ear
-*.zip
-*.rar
-
-# virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
-hs_err_pid*
-
-.idea
\ No newline at end of file
+target/
+.clover/
+.idea/
+.settings/
+.attach_pid*
+.classpath
+.directory
+.patch
+.project
+.surefire-*
+.DS_Store
+*.iml
+*.ipr
+*.iws
+*.dat
+atlassian-ide-plugin.xml
+*dependency-reduced-pom.xml
+hazelcast/src/main/java/com/hazelcast/instance/GeneratedBuildProperties.java
```

Comment:
What is this file?

Reply:
I copied the .gitignore file from main Hazelcast repo. It remained from there, I will remove it.